index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tasty Eats</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+        integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="/JavaScript-capstone-2/style.css">
+</head>
+
+<body>
+    <header>
+        <!-- <img src= "assets/images/foodlogo.png" alt="logo"> -->
+        <img id="img" src="./src/assets/images/jumpy.gif" alt="logo" width="100" height="100">
```

Comment:
- [ ] **Required:** This logo works in the live demo but not in the local project. Kindly fix it. 

![image](https://user-images.githubusercontent.com/27709832/165499403-a7ecb669-f611-417b-9e61-ecaae7f640da.png)